samples/msal-node-samples/auth-code-pkce/src/types/index.ts

```diff
@@ -0,0 +1,4 @@
+import { Request} from "express";
+import { Session } from "express-session";
+
+export type RequestWithPKCE = Request & { session: Session & { pkceCodes: { challengeMethod: string, challenge?: string, verifier?: string } }};
```

Comment:
Have you considered [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) instead? Seems to be favored [in some situations](https://stackoverflow.com/questions/37377731/extend-express-request-object-using-typescript), but this is fine too.